x-pack/plugins/actions/server/create_execute_function.ts

```diff
@@ -187,6 +229,37 @@ export function createEphemeralExecutionEnqueuerFunction({
   };
 }
 
+export async function hasReachedTheQueuedActionsLimit(
+  taskManager: TaskManagerStartContract,
+  configurationUtilities: ActionsConfigurationUtilities,
+  numberOfActions: number
+) {
+  const limit = configurationUtilities.getMaxQueued();
+  const { docs: tasks } = await taskManager.fetch({
```

Comment:
The default number of docs returned by ES when size is not set is 10, so just looking at the length of the docs returned will always show max of 10 queued. This should probably be updated to use `taskManager.aggregate` and to parse the `hits.total` value returned from that response in order to get the actual number of tasks in the queue

Reply:
Resolved in this commit [e549fd2](https://github.com/elastic/kibana/pull/164632/commits/e549fd29691687627ce2e39f27f671c35bd45e18)